bigbluebutton-html5/imports/startup/client/intl.js

```diff
@@ -9,6 +9,8 @@ const defaultProps = {
   locale: 'en',
 };
 
+let appLocale;
```

Comment:
Is there a reason why you don't keep this variable in the state?
It can be initialized in componentWillMount and updated in componentWillUpdate in a similar way as it is now.